app/scripts/react-directives/users/usersPage.jsx

```
@@ -0,0 +1,71 @@
+          <thead>
+            <tr>
+              <th>{t('users.labels.login')}</th>
+              <th>{t('users.labels.type')}</th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {store.users.map((user) => (
+              <tr key={user.id}>
+                <td style={{ verticalAlign: 'middle' }}>{user.login}</td>
+                <td style={{ verticalAlign: 'middle' }}>{t('users.labels.' + user.type)}</td>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)